infra/email_receipt_inbox/lambdas/parsers/parse_apple.py

```
@@ -0,0 +1,311 @@
+def flat(txt):
+    return re.sub(r"\s+", " ", txt)
+
+
+def money(m):
+    return float(m.replace(",", "")) if m else None
+
+
+def header_date_iso(msg):
+    try:
+        return msg["date"].datetime.date().isoformat()
+    except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)